src/zzz_od/application/hollow_zero/lost_void/lost_void_app.py

```
@@ -261,7 +264,76 @@ def matrix_select_team(self) -> OperationRoundResult:
+        # 3. 选人
+        for agent_loc in range(len(agent_page_match_list)):
+            self.swipe_multiple_times(max_swipe_times, 0.2, page_end, page_start)
+            self.swipe_multiple_times(agent_page_match_list[agent_loc][0], 0.2, page_start, page_end)
+            self.ctx.controller.click(agent_page_match_list[agent_loc][1])
+            time.sleep(0.5)
+
+        return self.round_success()
+
+    # 滑动x次
+    def swipe_multiple_times(self, swipe_num, wait, start, end):
+        for page in range(swipe_num):
```

Ruff (0.15.1)

[warning] Loop control variable `page` not used within loop body

Rename unused `page` to `_page`

(B007)